code/Model/SiteTree.php

```diff
@@ -2105,147 +2124,135 @@ public function VirtualPages()
      */
     public function getCMSFields()
     {
-        $dependentNote = '';
-        $dependentTable = new LiteralField('DependentNote', '<p></p>');
+        $this->beforeUpdateCMSFields(function (FieldList $fields) {
+            $dependentNote = '';
+            $dependentTable = new LiteralField('DependentNote', '<p></p>');
```

Comment:
I didn't add this code - and broadly changing from `new` to `create` should be done as part of https://github.com/silverstripe/silverstripe-framework/issues/10269.

I'll make the changes here just to avoid ping pong.

Reply:
Done